lib/src/text-input/TextInput.test.js

```diff
@@ -30,27 +45,7 @@ const countries = [
 ];
 
 describe("TextInput component tests", () => {
-  test("Renders with correct label", () => {
-    const { getByText } = render(<DxcTextInput label="Example label" />);
-    expect(getByText("Example label")).toBeTruthy();
-  });
-  test("Renders with correct label and helper text", () => {
-    const { getByText } = render(<DxcTextInput label="Example label" helperText="Example helper text" />);
-    expect(getByText("Example label")).toBeTruthy();
-    expect(getByText("Example helper text")).toBeTruthy();
-  });
-  test("Renders with correct label and optional", () => {
-    const { getByText } = render(<DxcTextInput label="Example label" helperText="Example helper text" optional />);
-    expect(getByText("Example label")).toBeTruthy();
-    expect(getByText("(Optional)")).toBeTruthy();
-    expect(getByText("Example helper text")).toBeTruthy();
-  });
-  test("Renders with correct placeholder", () => {
-    const { getByRole } = render(<DxcTextInput label="Example label" placeholder="Placeholder" />);
-    const input = getByRole("textbox");
-    expect(input.getAttribute("placeholder")).toBe("Placeholder");
-  });
```

Comment:
These tests are all related to visual elements and their presence. Since we now have visual regression testing, this kind of test is not valuable anymore.

Reply:
Maybe the one that tests `placeholder` should be maintained, or you can add it to the tests related to aria accessibility attributes.

Reply:
Why? `const input = getByRole("textbox");` is in every other test, and the placeholder is also a visual text, why should we keep it?